pallets/gear/src/benchmarking/syscalls.rs

```diff
@@ -243,8 +244,18 @@ where
     T: Config,
     T::AccountId: Origin,
 {
-    fn prepare_handle(code: WasmModule<T>, value: u32) -> Result<Exec<T>, &'static str> {
+    fn prepare_handle<PreProcessFn>(
+        code: WasmModule<T>,
+        value: u32,
+        mut pre_process: PreProcessFn,
```

Comment:
Just recommendation : `prepare_handle` is a function to reduce code size and therefor make code read a little easy. In cases when you need some extra logic , you can use `prepare` function instead.

Reply:
I think is better to use `prepare` instead of making `pre_process` argument.

Reply:
I don't see the point of `prepare_handle` fn at all.

Reply:
Agree with @grishasobol here

Reply:
And even if we come to solution with `pre_process`, the most common way is to throw Option here, instead of each time call empty closure